examples/Directory.Build.props

```diff
@@ -0,0 +1,3 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'opentelemetry-dotnet-contrib.sln'))\build\Common.nonprod.props" />
+</Project>
```

Comment:
would be nice to have a README.md in this directory that explains the example directory structure. For example, why there is no `server-core` while two flavors of a client?

Reply:
Simple answer, the WCF server bits aren't available on .NET Core 😄 But I'll add a doc.